packages/amplify-graphql-auth-transformer/src/resolvers/query.ts

```diff
@@ -303,7 +312,8 @@ export const generateAuthExpressionForRelationQuery = (
   fields: ReadonlyArray<FieldDefinitionNode>,
   primaryFieldMap: RelationalPrimaryMapConfig,
 ) => {
-  const { cogntoStaticRoles, cognitoDynamicRoles, oidcStaticRoles, oidcDynamicRoles, apiKeyRoles, iamRoles, lambdaRoles } = splitRoles(roles);
+  const { cogntoStaticRoles, cognitoDynamicRoles, oidcStaticRoles, oidcDynamicRoles, apiKeyRoles, iamRoles, lambdaRoles } =
```

Comment:
Doesn't look related to this PR, but is `cogntoStaticRoles` a typo?

Reply:
Yup 😅 I can address that here as well